src/authentication/authentication.controller.ts

```diff
@@ -52,10 +52,8 @@ export class AuthenticationController {
   @ApiOkResponse({
     description: 'Authenticates a user using EPA CDX Services',
   })
-  signOut(
-    @Body() credentials: ValidateTokenDTO,
-    @ClientIP() clientIp: string,
-  ): Promise<void> {
-    return this.service.signOut(credentials.token, clientIp);
+  signOut(@Req() req: Request, @ClientIP() clientIp: string): Promise<void> {
+    const token = req.headers['authorization'].split(' ')[1];
+    return this.service.signOut(token, clientIp);
   }
```

Comment:
not required but if you find yourself needing to retrieve the token from the request multiple times may want to consider creating an @AuthToken decorator similar to the @ClientIP & @CurrentUser decorators that encapsulates this logic